CIP-0086/README.md

```diff
@@ -342,11 +335,8 @@ and each inner array of `values` must be equal in length to `field_paths`.
 }
 ```
 
-A tabular metadata update transaction must
-only send ADA from the oracle update address to itself
-and must not mint any tokens,
-but it may contain any number of inputs and outputs.
-Otherwise, it is ignored for the purposes of token metadata.
+A tabular metadata update transaction may contain any inputs and outputs,
+as these are ignored for the purposes of updating token metadata.
```

Comment:
```suggestion
A tabular metadata update transaction must be signed
by the signing key of the oracle update address,
but it may contain any inputs and outputs,
as these are ignored for the purposes of updating token metadata.
```